packages/symreg/src/symreg/genetic_programming.py

```diff
@@ -114,61 +115,35 @@ def evalSymbReg(self, individual, points):
         :return: float, the mean squared error of the individual plus a size penalty if applicable
         """
         func = self.toolbox.compile(expr=individual)
-        sqerrors = ((func(x) - y) ** 2 for x, y in points)
+        sqerrors = (round(func(x) - y, 3) ** 2 for x, y in points)
```

Comment:
I'm rounding off 3 decimal points because I noticed the fitness values were getting unnecessarily precise -- I observed  a fitness value around`~1e-34`. At that point, we might as well treat that as a 0.

The 3 decimal point round-off is an arbitrary choice and I'm open to suggestions on how much precision we need